recipes/ngspicesimserver/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "NgspiceSimServer" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NyanCAD/NgspiceSimServer/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: debea3105f5828aa5da9910b26e9a83e880323d44b62eb267f0f3c97ade27450
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [not win]
+    - ninja  # [win]
+    - pkgconfig
+  host:
+    - capnproto
+    - ngspice-lib
+    - openssl 1.1*
+  run:
+    - ngspice-lib
+    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx and x86_64]
+
+test:
+  commands:
+    - $PREFIX/bin/NgspiceSimServer --help  # [not win]
```

Comment:
These executables should be in PATH so I guess you don't need to call the directly with the full path. Aka `NgspiceSimServer --help` should work in all platforms.

Reply:
Hey that seems to have worked :)